pydatalab/pydatalab/routes/v0_1/users.py

```diff
@@ -0,0 +1,16 @@
+from bson import ObjectId
+from flask import Blueprint, jsonify, request
+
+from pydatalab.mongo import flask_mongo
+
+user = Blueprint("users", __name__)
+
+
+@user.route("/users/<user_id>", methods=["PATCH"])
+def save_user(user_id):
+    user_name = request.json.get("data", {}).get("user_name")
```

Comment:
This needs to be locked down somewhat; probably a user is the only person that should be allowed to update their display name, maybe also admins (in case someone goes rogue and changes their name to something offensive...). We should therefore add a check like:

```suggestion
    user_name = request.json.get("data", {}).get("user_name")
    
    if not current_user.authenticated:
        return jsonify(status="error"), 401
    
    if current_user.id != user_id and current_user.role != "admin":
        return jsonify(status="error"), 403
    
```